src/components/user/user.service.ts

```diff
@@ -209,58 +207,131 @@ export class UserService {
             canReadCountry: true,
             canDeleteOwnUser: true,
             canDeleteLocation: true,
-            owningOrgId: "Seed Company",
-            isAdmin: true
-          })
-          -[:email {active: true, createdAt: datetime()}]->
-          (email:EmailAddress:Property {
+            owningOrgId: 'Seed Company',
+            isAdmin: true,
+          }),
+          relation('out', '', 'email', {
+            active: true,
+            createdAt: 'datetime()',
+          }),
+          node('email', 'EmailAddress:Property', {
             active: true,
-            value: $email,
-            createdAt: datetime()
+            value: input.email,
+            createdAt: 'datetime()',
```

Comment:
This is interpreted as a literal string with the query builder.
```suggestion
            createdAt: DateTime.local().toNeo4JDateTime(),
```
Feel free to make variable to not repeat yourself.

Reply:
Changes applied!